_posts/2021-06-30-geoextent-presented-at-the-2021-Earth-Cube-meeting.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "geoextent presented at the 2021 Earth Cube"
+author: "Sebastian Garzón"
+categories:
+  - geoextent
+  - geospatial 
+  - metadata 
+  - scholarly publishing 
+  - reproducibility
+---
+
+The Python library [geoextent](https://o2r.info/geoextent/) by the o2r project team was selected for presentation at the [2021 Earth Cube annual meeting](https://www.earthcube.org/2021-earthcube-annual-meeting) in the peer-reviewed notebooks session.
+In this blog post, student assistant Sebastian reports from the event.
+
+<!--more-->
+
+## geoextent presented at the 2021 Earth Cube
+
+Notebooks as a scholarly object, database interoperability, FAIR workflows, connecting data and code, and tools for geosciences research are some of the topics discussed at the [2021 EarthCube annual meeting](https://www.earthcube.org/2021-earthcube-annual-meeting).
+At the event, o2r team members [Sebastian](https://orcid.org/0000-0002-8335-9312) and [Daniel](https://orcid.org/0000-0002-0024-5046) presented [geoextent](https://github.com/o2r-project/geoextent), a Python library designed to extract temporal and spatial extent from data files.
+
+We presented the librry as part of the [2nd call for Notebooks](https://www.earthcube.org/2021-earthcube-annual-meeting) for a digital proceedings of the EarthCube Annual Meeting following the increased interest of the geosciences research community on reproducible workflows. 
+
+# Exploring Research Data Repositories with geoextent.
+
+> Sebastian Garzón and Nüst, Daniel. 2021. **Exploring Research Data Repositories with geoextent**. EarthCube Annual Meeting. [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/earthcube2021/ec21_garzon_etal/HEAD)
+
+The [notebook](https://mybinder.org/v2/gh/earthcube2021/ec21_garzon_etal/HEAD), accessible through [Binder](https://mybinder.org/), includes an introduction of geoextent's usage and a case study where we explored more than 300 [Zenodo](https://zenodo.org) repositories (over 25.000 files!) with geoextent.
+An initial exploration of Zenodo's API showed that spatial metadata is rarely available, difficulting data integration and discovery.
+The objective of our case study was to verify if we can increase the current percentage of repositories with geospatial information on [Zenodo](https://zenodo.org) by using geoextent.
+
+[![](../public/images/2021-06-30-geoextent-presented-at-the-2021-Earth-Cube-meeting/Screenshot-presentation-zenodo-api.png)](../public/images/2021-06-30-geoextent-presented-at-the-2021-Earth-Cube-meeting/Screenshot-presentation-zenodo-api.png)  
```

Comment:
Please check the other posts how to create the image URL - I'm not sure the `../` is correct.